### PR TITLE
Anomali threatstream build ioc json

### PR DESCRIPTION
**Pull request by sapirshuker**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-5636

## Description
Builds A JSON array based on the values provided by the user for the 'threatstream-import-indicator-without-approval' command.

## Minimum version of Cortex XSOAR
- [X] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X] No

## Must have
- [X] Tests
- [X] Documentation

### CODE REVIEW

File: Packs/Anomali_ThreatStream/Scripts/ThreatstreamBuildIocImportJson/ThreatstreamBuildIocImportJson_test.py [hunk 1]
```diff
@@ -0,0 +1,126 @@
+import json
+import io
+import pytest
+import demistomock as demisto
+from CommonServerPython import DemistoException
+
+
+def util_load_json(path):
```
Comment (Shellyber) [inline Packs/Anomali_ThreatStream/Scripts/ThreatstreamBuildIocImportJson/ThreatstreamBuildIocImportJson_test.py:8]: Do you use it somewhere? if not remove

File: Packs/Anomali_ThreatStream/Scripts/ThreatstreamBuildIocImportJson/ThreatstreamBuildIocImportJson_test.py [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    with io.open(path, mode='r', encoding='utf-8') as f:
+        return json.loads(f.read())
+
+
+@pytest.mark.parametrize('args, indicators_types, expected_result', [
```
Comment (Shellyber) [inline Packs/Anomali_ThreatStream/Scripts/ThreatstreamBuildIocImportJson/ThreatstreamBuildIocImportJson_test.py:13]: Please add a case with comma-separated values as well

File: Packs/Anomali_ThreatStream/Scripts/ThreatstreamBuildIocImportJson/ThreatstreamBuildIocImportJson_test.py [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+
+@pytest.mark.parametrize('email_list, md5_list, ip_list, url_list, domain_list, expected_result', [
+    (['email'], [], [], [], [], 'Invalid indicators values: email'),
+    ([], ['111'], [], [], [], 'Invalid indicators values: 111'),
+    ([], [], ['1.1'], [], [], 'Invalid indicators values: 1.1'),
+    ([], [], [], ['url'], [], 'Invalid indicators values: url'),
+    ([], [], [], [], ['domain'], 'Invalid indicators values: domain'),
+])
+def test_validate_indicators(email_list, md5_list, ip_list, url_list, domain_list, expected_result):
```
Comment (Shellyber) [inline Packs/Anomali_ThreatStream/Scripts/ThreatstreamBuildIocImportJson/ThreatstreamBuildIocImportJson_test.py:59]: Add also a test for valid indicators

File: Packs/Anomali_ThreatStream/Scripts/ThreatstreamBuildIocImportJson/ThreatstreamBuildIocImportJson_test.py [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+      "url_values": 'https://test.com'},
+     {'email': 'mal_email', 'md5': 'mal_md5', 'ip': 'mal_ip', 'url': 'mal_url', 'domain': 'mal_domain'},
+     [{'value': 'test@test.com', 'itype': 'mal_email'},
+      {'value': '00000000000000000000000000000000', 'itype': 'mal_md5'},
+      {'value': '1.1.1.1', 'itype': 'mal_ip'},
+      {'value': 'https://test.com', 'itype': 'mal_url'},
+      {'value': 'my.domain1.com', 'itype': 'mal_domain'}])
+])
+def test_get_indicators_from_user(args, indicators_types, expected_result):
+    """Tests get_indicators_from_user function.
```
Comment (Shellyber) [inline Packs/Anomali_ThreatStream/Scripts/ThreatstreamBuildIocImportJson/ThreatstreamBuildIocImportJson_test.py:43]: Change docstrings to the format we use `given, when, then`

**Review by Shellyber — CHANGES_REQUESTED** [review]
Great job!

**Review by Shellyber — APPROVED** [review]

**Comment by ShirleyDenkberg** [comment]
@Shellyber Doc review completed.

**Pull request merged**